posts/fall2023/hey-future-interns/index.md

```diff
@@ -0,0 +1,97 @@
+---
+date: 2023-09-01
+title: "Hey Future Interns!"
+subtitle: "An Overview of the ACM Hack Internship Program"
+author: "Nathan Zhang and James Wu"
+---
```

Comment:
I think the subtitle should include "... with the 2022-2023 Hack intern class" at the end, or something similar (that kind of gives a nod to the fact it's more of us interviewing the interns than talking about logistical structure).

If it's too long to append, maybe something like "Life at Hack, with the 2022-2023 intern class"

Reply:
An Overview of the ACM Hack Internship Program as Told by Our Most Recent Interns
^ how's this

Reply:
Yep sounds good 👍